assets/lang/sklmessages_es_ES.properties

```diff
@@ -51,48 +53,50 @@ login.existing.logout=Desconectar
 editor.latest=Usar la \u00faltima Versi\u00f3n
 editor.exe=Ejecutable
 editor.jvm=Par\u00e1metros de JVM
 editor.copy=Copia de
-editor.hide=Ocultar el launcher y reabrirlo cuando el juego cierre
-editor.close=Cerrar el launcher cuando el juego comience
-editor.keep=Mantener el launcher abierto
-editor.profilename=Nombre del perfil
-editor.alert.title=Confirmaci\u00f3n del perfil
-editor.alert.text=\u00bfSeguro que quieres borrar este perfil?
-editor.compatibility=Modo de compatibilidad
-editor.compatibility.tooltip=Los "skins" no funcionan en este modo, pero algunos paquetes lo necesitan
+editor.hide=Ocultar el Launcher y reabrirlo cuando el Juego cierre
+editor.close=Cerrar el Launcher cuando el Juego comience
+editor.keep=Mantener el Launcher abierto
+editor.profilename=Nombre del Perfil
+editor.alert.title=Confirmaci\u00f3n del Perfil
+editor.alert.text=\u00bfSeguro que quieres borrar este Perfil?
+editor.compatibility=Modo de Compatibilidad
+editor.compatibility.tooltip=Los "Skins" no funcionan en este Modo, pero algunos Paquetes lo necesitan
 
-settings.title=Configuraci\u00f3n del launcher
+settings.title=Configuraci\u00f3n del Launcher
 settings.lang=Idioma
-settings.token=C\u00f3digo de acceso especial
+settings.lang.help=\u00bfTu Idioma no est\u00e1 en la Lista?
+settings.lang.link=\u00a1Ay\u00fadanos a traducir!
+settings.token=C\u00f3digo de Acceso especial
 settings.save=Aplicar
 settings.cancel=Cancelar
 settings.theme=Tema
 settings.theme.dark=Tema oscuro
 settings.theme.light=Tema claro
 settings.alert.title=Informaci\u00f3n
-settings.alert.text=Necesitas reiniciar el launcher para aplicar los cambios
+settings.alert.text=Necesitas reiniciar el Launcher para aplicar los Cambios
 
-crash.sorry=Vaya, parece que el juego se ha ca\u00eddo, disculpa las molestias :(
-crash.vanilla=Usando magia y amor, hemos podido recolectar algunos detalles de la ca\u00edda para investigarla lo m\u00e1s pronto posible. Puedes ver el reporte completo m\u00e1s abajo
-crash.modded=Detectamos que el juego podr\u00eda estar modificado, asi que no podemos recibir el reporte de ca\u00edda. De todos modos, si est\u00c1s usando mods, env\u00edale este reporte a los autores del mod para que le echen un vistazo
-crash.open=Abrir archivo de reporte
+crash.sorry=Vaya, parece que el Juego se ha ca\u00eddo. Disculpa las Molestias :(
+crash.vanilla=Usando Magia y Amor, hemos podido recolectar algunos Detalles de la ca\u00edda para investigarla lo m\u00e1s pronto posible. Puedes ver el Reporte completo m\u00e1s abajo
+crash.modded=Detectamos que el Juego podr\u00eda estar modificado, asi que no podemos recibir el Reporte de ca\u00edda. De todos Modos, si est\u00c1s usando Mods, env\u00edale este Reporte a los Autores del Mod para que le echen un Vistazo
+crash.open=Abrir Archivo de Reporte
 
 console.title=Consola %s
 
-about.translation=Traducido
-about.translation.authors=atesin, Dozerly
+about.translation=Traducido por:
+about.translation.authors=atesin, dozerly
```

Comment:
This nickname should remain unchanged. @Dozerly's profile on GitHub shows that the first letter in his nickname is in upper case.

Reply:
I generally don't care about how my name shows, in terms of uppercase/lowercase, to be honest.

Reply:
do whatever you want... i thougth to delete him even

and where are the rules about remain "unchanged" the previous work without asking the original author?

i think the main responsibile for these misunderstoods is skmedix for allow anybody to enter and let overwrite current files without any control... i hope you all learn about this

do whatever you want, this is not funny for me anymore

Reply:
the last thing.....   i changed his name because i think he is not more important than me in this work

i spent DAYS quering pages as RAE (spanish language royal academy in spanish), online dictionaries and grammar pages, etc, to do the best quality translation i could, to do the best launcher in the internet...  the only thing dozerly did when he came is to change some letters (and mess a couple) without anybody noticing me

i feel it like a misrespect by all staff (and still) (from skmedix to below) to all the time and effort invested in helping the project ...  he did not worked more than me, so he is not more important than me, so his name would not be bigger or longer than mine, that is what i changed .... however until nobody did organize workflows before, in this case you came immediately to complain me about this

i felt the need to say it... now please don't bother me anymore with messages about it

Reply:
So if you are undoing changes made by him, just remove his name completely, without worrying about this. If none of his changes are here, he's no longer a translator.

Reply:
I suggested changes, because I just noticed that you had changed his nickname without giving a reason. Spanish language is not my native language, so i didn't comment other parts of your work, as you could notice.

In my opinion if you are undoing changes made by him, just remove his name completely, without worrying about this. If none of his changes are here, he's no longer a translator.

If you felt better, I would approve your changes :)

Reply:
hi greg ...  it is likely spanish were not your native language because as i saw your task is mainly sort files...  is not your native language as english is not mine, but however everybody understand us speaking/writing it

i already gave my reasons, i thougth it is worth to give them

i think sklauncher is currently one of the best launchers on the internet... i told when i start to contribute, maybe a couple years ago, i started by having long mail and chat conversations with skmedix with suggestions, opinions, etc... i was very happy when skmedix started to develop skl v3, and even more when i offer him my help translating and he agreed... i already had developed some minecraft server mods and played in a lot of different servers, i also have some java skills but seems not as much as skmedix ones

in my opinion, you have a terrific team, have many assorted skills needed to do a quality product, but however as maybe you are young, you lack the social and human management skills .... _**i was the first spanish translator**_ and as such i deserve respect, you should at least had notify me _**before**_ had commited changes

i sincerely appreciate your discharges, but the result is not as instant as we would like ... the damage was done, i already felt misrespected and i'm afraid i need some time to cure myself ... in the meanwhile i hope you find some other contributor that also agree to translate spanish files passionately (maybe even dozerly)

don't do it again